src/fs/bootstrap_record.hh

```diff
@@ -0,0 +1,83 @@
+/*
+ * This file is open source software, licensed to you under the terms
+ * of the Apache License, Version 2.0 (the "License").  See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership.  You may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/*
+ * Copyright (C) 2019 ScyllaDB
+ */
+
+#pragma once
+
+#include "cluster.hh"
+#include "inode.hh"
+
+#include <seastar/core/reactor.hh>
+#include <seastar/fs/block_device.hh>
+
+namespace seastar::fs {
+
+class invalid_bootstrap_record : public std::runtime_error {
+public:
+    invalid_bootstrap_record(const char* msg) : std::runtime_error(msg) {}
```

Comment:
I don't like that, because std::runtime_error takes a std::string as a parameter and here it is somewhat restricted. Maybe use a std::exception or allow for std::string?